qml/components/MessageListViewItem.qml

```diff
@@ -136,12 +137,12 @@ ListItem {
             messageBackground.isUnread = index > chatModel.getLastReadMessageIndex();
         }
         onLastReadSentMessageUpdated: {
-            console.log("[ChatModel] Messages in this chat were read, new last read: " + lastReadSentIndex + ", updating description for index " + index + ", status: " + (index <= lastReadSentIndex));
+            Debug.log("[ChatModel] Messages in this chat were read, new last read: " + lastReadSentIndex + ", updating description for index " + index + ", status: " + (index <= lastReadSentIndex));
```

Comment:
Would this work too?
```JavaScript
Debug.log("[ChatModel] Messages in this chat were read, new last read:", lastReadSentIndex, ", updating description for index", index, ", status:", (index <= lastReadSentIndex));
```
That would minimize runtime penalty to a minimum by letting `console.log()` (if it ends up being invoked) to do the concatenation.

Reply:
Good Idea & done.